tardis/plasma/tests/test_tardis_model_density_config.py

```diff
@@ -30,12 +32,14 @@ def test_electron_densities(raw_plasma):
     assert_almost_equal(raw_plasma.electron_densities[3], 2.6e14)
 
 
-def test_isotope_number_densities(raw_plasma):
+def test_isotope_number_densities(raw_model):
     assert_almost_equal(
-        raw_plasma.isotope_number_density.loc[(28, 56), 0], 9688803936.317898
+        raw_model.composition.isotopic_number_density.loc[(28, 56), 0],
```

Comment:
@andrewfullard this test will fail as you are testing pre-decay. The new code is decaying when it loads.